### PR TITLE
Add new graphical representations to LanguageModel 

### PR DESCRIPTION
**Pull request by VicenteAR**
Added new methods to represent the lattice (using holoviews). A few problems are in order: 
- method `_plot()` inside `run` is not properly called (plot is not returned)
- I can't get the desired layout. The idea is showing a composition 2 cols + 1 col (is the solution presented [here](https://github.com/holoviz/holoviews/issues/91)?)

Signed-off-by: VicenteAR <vicente.arjona@hotmail.com>

### CODE REVIEW

**Comment by VicenteAR** [comment]
Fixed problems: 
- method `_plot()` inside `run` is not properly called (plot is not returned) => inside `_plot`, used __return display(...)__. This correctly shows the image. 
- I can't get the desired layout. The idea is showing a composition 2 cols + 1 col => used __panel.Column()__ and __panel.Row()__ combination. 


==================================================================
Style problem: command `display` is not recognize. Which module is needed?

File: language_competition/models.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
 """Module containing  custom classes to study different language competition models."""
 from typing import Optional, Tuple, Union
 
+from bokeh.models import FixedTicker
+import holoviews as hv
+from holoviews import opts
 import matplotlib as mpl
```
Comment (Guillemdb) [inline language_competition/models.py]:
```suggestion

```

File: language_competition/models.py [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 import matplotlib.pyplot as plt
```
Comment (Guillemdb) [inline language_competition/models.py]:
```suggestion
```

**Review by Guillemdb — CHANGES_REQUESTED** [review]
Please remove the `matplotlib` imports before merging.

File: language_competition/models.py [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
 import numpy as np
+import panel as pn
```
Comment (Guillemdb) [inline language_competition/models.py:10]:
```suggestion
from IPython.core.display import display

```

**Comment by VicenteAR** [comment]
Fixed problems. Removed matplotlib libraries. Changed `_repr_`: moved from `matplotlib` to `holoviews`.

**Pull request merged**